### PR TITLE
Add all model references.

### PR DESCRIPTION
**Pull request by phuclh**
Add all model references to support.php

### CODE REVIEW

**Comment by drewroberts** [comment]
It may be easier to have these ordered alphabetically.

There are a few missing at a first glance:

- tag
- topic
- page
- post
- series

They are not really referenced by other packages so they may not be necessary.

**Comment by drewroberts** [comment]
Should we also drop them into a `model_class` section of the config?

**Comment by phuclh** [comment]
> Should we also drop them into a `model_class` section of the config?

Ok I did, I removed it to run the `ksort` function and forgot to put it back. Thank you for reminding me.

File: src/SupportServiceProvider.php [hunk 1]
```diff
@@ -17,7 +17,7 @@ public function configurePackage(Package $package): void
          */
         $package
             ->name('support')
-            ->hasConfigFile()
```
Comment (drewroberts) [inline src/SupportServiceProvider.php:20]: I'm not sure if we can pass the name to this or if we need to handle it differently.

Comment (drewroberts) [reply src/SupportServiceProvider.php:20]: It looks like it accepts the name of the config file. The tests are passing! ✅

**Review by drewroberts — APPROVED** [review]
Thank you, Phuc! 🔱

**Comment by drewroberts** [comment]
Issue #21

**Review by joshtorres — COMMENTED** [review]
Looks good

**Pull request merged**